Services/Mail/test/ilMailTest.php

```diff
@@ -177,4 +191,369 @@ static function (string $login) use ($loginToIdMap) : int {
 
         ilMimeMail::setDefaultTransport($oldTransport);
     }
+
+    public function testGetMailObjectReferenceId() : void
+    {
+        $refId = 364;
+        $instance = $this->create($refId);
+
+        $this->assertEquals($refId, $instance->getMailObjectReferenceId());
+    }
+
+    public function testFormatNamesForOutput() : void
+    {
+        $instance = $this->create();
+
+        $this->assertEquals('', $instance->formatNamesForOutput(''));
```

Comment:
Hmmm, according to `\ilMail::formatNamesForOutput` a translation of `not_available` should be returned if the input string is empty. I wonder why `''` is expected here (and why this test works).

Reply:
Hmmm, IMO the test does not make sense in this case :-). I suggest either to test real scenarios oder just remove this test :-).